src/reporting/dasmon/models.py

```diff
@@ -22,8 +22,8 @@ class StatusVariable(models.Model):
     """
         Table containing key-value pairs from the DASMON
     """
-    instrument_id = models.ForeignKey(Instrument)
-    key_id = models.ForeignKey(Parameter)
+    instrument_id = models.ForeignKey(Instrument, on_delete=models.DO_NOTHING)
+    key_id = models.ForeignKey(Parameter, on_delete=models.DO_NOTHING)
```

Comment:
Pasting this here from slack just so its tied to the pr for visibility:
> the on_delete param is required, I set it to do nothing as there was no specified behavior I'm aware of.  The other option is to cascade, which would delete from the foreign tables if the current entry was deleted.  I think do nothing is safer as it doesnt remove any data.  If someone has insight into the data relationship we should ask them their opinion.

Reply:
Are you sure it isn't what happens to this entry when the foreign key is deleted rather than what happens to the foreign key when this entry is deleted? A link to documentation would help tremendously. What I found was [this blog post](https://jilles.me/django-foreignkeys-on_delete-handlers/).

Reply:
[This doc page](https://docs.djangoproject.com/en/4.0/ref/models/fields/) describes what each on_delete does, ctrl+f "ForeignKey.on_delete"

Reply:
[Link to the interesting bits](https://code.ornl.gov/sns-hfir-scse/imaging/imaging/-/issues/93)

Reply:
(you may have pasted the wrong link?)

Reply:
Reading [this tutorial](https://www.mysqltutorial.org/mysql-on-delete-cascade/) it looks like the behavior you are defining is what happens to the rows in *this table* if the `ForeignKey` referenced in a different table is deleted. In other words, if `CASCADE` is chosen and someone deletes the `Instrument` in the other table, delete all rows that reference it in this table. `DO_NOTHING` will leave this table in an invalid state. 

It looks like [django v1.8 had a default behavior of `CASCADE`](https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.ForeignKey.on_delete). Assuming that is the same default as v1.6, we should specify that option in the "new" code.